tests/js/spec/views/accountSecurity.spec.jsx

```diff
@@ -91,22 +97,39 @@ describe('AccountSecurity', function() {
     );
     expect(wrapper.find('AuthenticatorStatus').prop('enabled')).toBe(true);
 
+    // next authenticators request should have totp disabled
+    const authenticatorsMock = Client.addMockResponse({
+      url: ENDPOINT,
+      body: [
+        TestStubs.Authenticators().Totp({
+          isEnrolled: false,
+          authId: '15',
+          configureButton: 'Info',
+        }),
+      ],
+    });
+
     // This will open confirm modal
     wrapper.find('Button[icon="icon-trash"]').simulate('click');
+
     // Confirm
     wrapper
       .find('Modal Button')
       .last()
       .simulate('click');
 
+    await tick();
+    wrapper.update();
+
     expect(deleteMock).toHaveBeenCalled();
 
-    setTimeout(() => {
-      wrapper.update();
-      expect(wrapper.find('AuthenticatorStatus').prop('enabled')).toBe(false);
```

Comment:
This was never being called because of the `setTimeout`. The `setTimeout` makes the test case async, but because it was not marked as an async function (or defined with a `done` callback), the test runner assumes the test is synchronous.